docs/error-reporting/advanced/hydra/usage.md

```diff
@@ -0,0 +1,377 @@
+---
+id: usage
+title: Hydra Usage
+sidebar_label: Usage
+description: Hydra takes a snapshot generated by running Backtrace's ptrace tracer on an application and organizes the data in an easy to grok terminal interface.
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+Hydra is a tool that takes a snapshot generated by running our ptrace tracer on an application and organizes the data in an easy-to-understand terminal interface.
+
+Its main purpose is to help engineers focus on the most relevant information related to the root cause of a failure. The tool automatically selects the faulting frame, annotates suspect variables, and classifies the crash into a specific fault type (for example, null dereference, memory write). Hydra provides various commands and integrations to facilitate navigation through the application state, enabling faster access to more information and reducing tedious tasks.
+
+Let's dive deeper into how Hydra can help you eliminate the complex bugs that impact your production applications.
+
+## Overview
+
+### Running Hydra
+
+First things first: how do you run Hydra?
+
+```shell
+$ hydra
+```
+
+If you have configured coroner, you can also use:
+
+```shell
+$ coroner view
+```
+
+Running these commands instantly displays the exact root cause of the crash and, in some cases, even proposes a fix for you.
+
+What you actually see is a colorful ncurses interface (unless you pass the `-m` flag to hydra, in which case it is displayed in black and white). The interface reminds you of some of your favorite tools, such as htop and tig. So, how is this interface organized?
+
+### Understanding the Panes
+
+The interface is divided into several panes, each representing a different aspect of the application's state.
+
+The panes, in order, are:
+
+- Threads
+- Frames
+- Variables
+- Router
+
+We'll explain the Router pane in more detail later. For now, know that it contains various metadata about the application, including system information (memory/CPU usage), registers, kernel frames, and more. It also offers configurable integrations, such as source code management.
+
+Among the panes, the only one that remains constant is the Threads pane (well, almost constant, because you can still change how threads are displayed). The Frames, Variables, and Router panes dynamically update based on the selected context. We refer to these panes as context-aware. Frames are populated based on the presently selected thread, variables and registers depend on the current frame, and so on.
```

Comment:
```suggestion
Among the panes, the only one that remains constant is the Threads pane (well, almost constant because you can still change how threads are displayed). The Frames, Variables, and Router panes dynamically update based on the selected context. We refer to these panes as context-aware. Frames are populated based on the selected thread, Variables and Registers depend on the current frame, and more.
```